### PR TITLE
Fixed pip setup.py script

### LINKED ISSUE #2: patches/afl-patch.diff not distributed in shellphish-afl pip
`setup.py`  does not package `patches/afl-patch.diff`.  This results in the following error when attempting to `pip install shellphish-afl`:

```
Downloading/unpacking shellphish-afl
  Downloading shellphish-afl-1.0.tar.gz
  Running setup.py (path:/home/singlethink/.virtualenvs/driller/build/shellphish-afl/setup.py) egg_info for package shellphish-afl

Installing collected packages: shellphish-afl
  Running setup.py install for shellphish-afl
    Setting up AFL-other-arch
    Cloning into 'bin/afl-unix'...
    error: [Errno 2] No such file or directory: 'patches/afl-patch.diff'
    Complete output from command /home/singlethink/.virtualenvs/driller/bin/python -c "import setuptools, tokenize;__file__='/home/singlethink/.virtualenvs/driller/build/shellphish-afl/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-WJdyJC-record/install-record.txt --single-version-externally-managed --compile --install-headers /home/singlethink/.virtualenvs/driller/include/site/python2.7:
    running install

running build

Setting up AFL-other-arch

Cloning into 'bin/afl-unix'...

error: [Errno 2] No such file or directory: 'patches/afl-patch.diff'

----------------------------------------
Cleaning up...
Command /home/singlethink/.virtualenvs/driller/bin/python -c "import setuptools, tokenize;__file__='/home/singlethink/.virtualenvs/driller/build/shellphish-afl/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-WJdyJC-record/install-record.txt --single-version-externally-managed --compile --install-headers /home/singlethink/.virtualenvs/driller/include/site/python2.7 failed with error code 1 in /home/singlethink/.virtualenvs/driller/build/shellphish-afl
Storing debug log for failure in /home/singlethink/.pip/pip.log
```
## Temporary workaround

 `python setup.py install` from a clone of the `shellphish-afl` repo

### PR DESCRIPTION
**Pull request by CodeMaxx**
Fixes #2 as well

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -93,9 +102,12 @@ def run(self):
         _datafiles()
         _develop.run(self)
 
+get_patches()
```
Comment (zardus) [inline setup.py:105]: This doesn't seem to have any effect?

Comment (CodeMaxx) [reply setup.py:105]: This adds the `patches` folder to the tarball while `py setup.py sdist` packaging

Comment (CodeMaxx) [reply setup.py:105]: The return value isn't being used but the global `data_files` is being modified.

Comment (zardus) [reply setup.py:105]: Oh, got it. I was viewing the reduced diff.

Would you mind integrating `get_patches()` into `_datafiles()`? That'll keep this process cleaner.

Comment (CodeMaxx) [reply setup.py:105]: That was the first thing I tried, but `_datafiles()` is only being called during the `build` process after packaging. So the `patches` folder won't go into the tarball.

Comment (zardus) [reply setup.py:105]: Makes sense. Awesome stuff!

**Pull request merged**